specification/customerlockbox/resource-manager/Microsoft.CustomerLockbox/preview/2018-02-28-preview/customerlockbox.json

```diff
@@ -0,0 +1,359 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-02-28-preview",
+    "title": "Customer Lockbox",
+    "description": "Azure Customer Lockbox API Reference"
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
```

Comment:
what about deleting a request? It can serve the purpose of cleaning up the meta data of all the requests that have been raised in the past. I am not implying that Delete cancels the pending request.

Reply:
Currently all requests are kept for as long as necessary for auditing purposes. Lockbox requests aren't really like normal resources in Azure so it's hard to reason about what we should and should not be deleting. I will bring this up with my team to think more about how we want to "clean things up."

Reply:
We will look into this along side a put/create story.